container-search/pom.xml

```diff
@@ -138,6 +138,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+      <scope>provided</scope>
```

Comment:
Couldn't this be compile, and then you could skip this dep in container-search-and-docproc?